release-calendar/src/client/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
This file was recently deleted to use `src/client/index.ejs` as a template instead.